shell/src/test/java/alluxio/cli/fsadmin/command/SummaryCommandTest.java

```diff
@@ -109,6 +111,10 @@ private void checkIfOutputValid() {
         "    Uptime: 143 day(s), 15 hour(s), 53 minute(s), and 32 second(s)",
         "    Version: testVersion",
         "    Safe Mode: false",
+        "    Zookeeper Enabled: true",
```

Comment:
good to mention that the addresses we show below this are the zookeeper addresses, something along the lines of "Zookeeper Addresses:"